packages/components/components/Select/Select.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from "react";
+import { Box, Label } from "..";
+import { styled } from "../../stitches.config";
+import Input from "../Input";
+
+// Container of dropdown UI elements.
+const DropDownContainer = styled("div", {
+  width: "100%"
+});
+
+// Topo Input element repurposed for the dropdown.
+const SelectInput = styled(Input, {
+  backgroundImage: `url("data:image/svg+xml;utf8,<svg fill='black' height='24' viewBox='0 0 24 24' width='24' xmlns='http://www.w3.org/2000/svg'><path d='M7 10l5 5 5-5z'/><path d='M0 0h24v24H0z' fill='none'/></svg>") !important`,
+  backgroundPosition: '99% !important',
+  backgroundRepeat: 'no-repeat !important'
+});
+
+// Container of dropdown list items.
+const ListBox = styled(Box, {
+  borderStyle: "solid",
+  borderWidth: "0 1px 1px 1px",
+  borderColor: "$forest",
+  color: "$forest",
+  fontSize: "$base",
+  fontFamily: "$sans"
+});
+
+// Styled list item.
+const ListItem = styled("li", {
+  listStyle: "none",
+  padding: "$1 $2 $1 $3",
+  backgroundColor: "$sand",
+  color: "$forest",
+  borderStyle: "solid",
+  borderWidth: "1px 0px 0px 0px",
+  borderColor: "$forest",
+  "&:hover": {
+    backgroundColor: "$forest",
+    color: "$sand"
+  }
+});
+
+//Select component props.
+export interface ISelectProps {
+  disabled: boolean,
+  options: {key: string, label: string}[],
+  label?: string,
+  description?: string;
+  required?: boolean;
+  error?: string;
+  placeholder?: string;
+  defaultValue?: string;
+  value?: string;
+  onChange?(key: string): void;
+  tabIndex?: number;
+}
+
+export const Select = (props: ISelectProps) => {
+
+  let initialValue = "";
```

Comment:
this could stay a `const` with coalescing

`const initialValue = props.value || props.defaultValue || ""`; 

could then do something similar with `options` and `displayValue` below

Reply:
Coalesced initialValue